core/src/main/java/com/scalar/db/storage/cassandra/Cassandra.java

```diff
@@ -46,11 +48,9 @@ public class Cassandra extends AbstractDistributedStorage {
   public Cassandra(DatabaseConfig config) {
     super(config);
 
-    if (config.isCrossPartitionScanFilteringEnabled()
-        || config.isCrossPartitionScanOrderingEnabled()) {
+    if (config.isCrossPartitionScanOrderingEnabled()) {
       throw new IllegalArgumentException(
-          CoreError.CASSANDRA_CROSS_PARTITION_SCAN_WITH_FILTERING_OR_ORDERING_NOT_SUPPORTED
```

Comment:
Ahh, we do not maintain old ones for reference, do we? OK, then I will remove them.

Reply:
I think we can remove old error messages.

@feeblefakie @josh-wong What do you think?

Reply:
Fixed in 580fae5. Note that, as discussed with Toshi, we need to maintain deleted messages to avoid reusing already deleted message IDs somehow, especially the most recent ones.